### PR TITLE
feat: feature kyverno adapters

### PR DESCRIPTION
**Pull request by pns-nirmata**

### CODE REVIEW

File: charts/enterprise-kyverno-operator/README.md [hunk 1]
```diff
@@ -93,4 +92,7 @@ helm uninstall -n enterprise-kyverno-operator enterprise-kyverno-operator
 | kyverno.image.pullPolicy | string | `"IfNotPresent"` | Kyverno Image pull policy |
 | kyverno.image.tag | string | `v1.9.1-n4k.nirmata.1` | Image tag (defaults to chart app version) |
 | kyverno.helm | object | `helm.rbac.serviceAccount.name=kyverno` | Free form yaml section with helm parameters in Kyverno chart. See all parameters [here](https://github.com/nirmata/kyverno-charts/tree/main/charts/nirmata#values) |
-| policies.policySets | list | `[{name: best-practices, type: helm, chartRepo: https://nirmata.github.io/kyverno-charts, chartName: best-practice-policies, version: 0.1.0}, {name: pod-security, type: helm, chartRepo: https://nirmata.github.io/kyverno-charts, chartName: pod-security-policies, version: 0.1.0}]` | Default policy sets to install along with operator |
+| policies.enableBestPractices | bool | `true` | Whether default best practices policy set is to be installed along with operator |
```
Comment (atulatnirmata) [inline charts/enterprise-kyverno-operator/README.md:95]: Should this be kyverno.policies.enableBestPractices?

File: charts/enterprise-kyverno-operator/README.md [hunk 2]
```diff
@@ -93,4 +92,7 @@ helm uninstall -n enterprise-kyverno-operator enterprise-kyverno-operator
...
+| policies.enablePodSecurity | bool | `true` | Whether default pod security policy set is to be installed along with operator |
```
Comment (atulatnirmata) [inline charts/enterprise-kyverno-operator/README.md:96]: same here?

File: charts/enterprise-kyverno-operator/values.yaml [hunk 3]
```diff
@@ -88,26 +86,35 @@ kyverno:
...
-    chartRepo: https://nirmata.github.io/kyverno-charts
-    chartName: best-practice-policies
-    version: 0.1.0
-  - name: pod-security
-    type: helm
-    chartRepo: https://nirmata.github.io/kyverno-charts
-    chartName: pod-security-policies
-    version: 0.1.0
+  enableBestPractices: true
+  enablePodSecurity: true
```
Comment (atulatnirmata) [inline charts/enterprise-kyverno-operator/values.yaml:98]: fix the trailing spaces github lint failure

**Review by atulatnirmata — APPROVED** [review]

File: charts/enterprise-kyverno-operator/README.md (hunk 1 above)
Comment (pns-nirmata) [reply charts/enterprise-kyverno-operator/README.md]: Actually, the policies section is no longer under kyverno. Plus Anusha suggested a further change to spec format. Added that now.

**Pull request merged**